controllers/config/overlays/kind/controllersconfig/cf_k8s_controllers_config.yaml

```diff
@@ -1,4 +1,5 @@
 kpackImageTag: gcr.io/cf-relint-greengrass/cf-k8s-controllers/kpack/beta
+
```

Comment:
Is this space intentional?

Reply:
definitely not..just a leftover crumb from previous hacking. I took it back out.